server/src/testFixtures/java/io/crate/test/utils/ConnectionTest.java

```diff
@@ -114,7 +114,7 @@ protected void initChannel(Channel channel) throws Exception {
             return ProbeResult.SSL_MISSING;
         }
         finally {
-            group.shutdownGracefully().sync();
+            group.shutdownGracefully(100, 1000, TimeUnit.MILLISECONDS).sync();
```

Comment:
Cant we disable the `quitePeriod` here completely? 
```suggestion
            group.shutdownGracefully(0, 5, TimeUnit.SECONDS).sync();
```

Reply:
> Cant we disable the `quitePeriod` here completely?

I put some small value just to ensure there no residual things to handle - but probably 0 is safe since it happens after 
`connectFuture.channel().closeFuture().sync();`

Reply:
Will squash and reword to something relevant